routes/mongo.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config();
+var express = require('express');
+var router = express.Router();
+const MongoClient = require('mongodb').MongoClient;
+const uri = "URIstring"
+const client = new MongoClient(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+router.get('/inputone', async (req, res) => {
+    try {
+        await client.connect();
```

Comment:
Are we closing the connection at some point?

Reply:
Good catch - will update all 3 and submit new PR